caffe2/python/optimizer_test.py

```diff
@@ -21,6 +23,19 @@ def check_optimizer(self, optimizer):
             np.testing.assert_allclose(np.array([1.0]), tensor, atol=1e-5)
 
 
+class TestMultiPrecisionSgd(OptimizerTestBase, TestCase):
+    def build_optimizer(self, model):
+        self._skip_gpu = False
+        return build_multi_precision_sgd(model, base_learning_rate=0.1)
+
+    def check_optimizer(self, optimizer):
```

Comment:
could you actually test that we at least have those params with a copy and conversion ops in place?